README.md

```
@@ -1,18 +1,20 @@
+</span>
 
 ## Project Structure
-Calculator is made to automatically load **Weapons** and **Drive Discs** from `Resources` folder.
-Agents are hard-coded, because it's easier to calculate relative stats that way.
-All configurations are in JSON format and named with IDs of the corresponding objects. These IDs can be found in
-[interknot-calculator/IDs.md](IDs.md) file.
+Agents, W-Engines (Weapons) and Drive Disc Sets configurations are all represented as Classes.
 
 ## How damage is being calculated
-*(short ver., long version can be found [here](https://github.com/Night-Sky-Studio/interknot-calculator/wiki/Damage-Calculation))*
+*(short ver., long version can be found [here](https://youtrack.interknot.space/articles/IKC-A-1))*
```

markdownlint-cli2 (0.23.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)